app/controllers/api/trainees/degrees_controller.rb

```diff
@@ -38,7 +38,7 @@ def update
 
       def destroy
         if degree.destroy
-          render(json: { data: serializer_class.new(degree).as_hash })
+          render(json: { data: serializer_class.new(trainee).as_hash })
```

Comment:
I think this isn't quite right because the `serializer_class` in this case will be the `DegreeSerializer`. Maybe do what the Placements controller does, and have an additional method:
```ruby
      def trainee_serializer_class
        Serializer.for(model: :trainee, version: version)
      end
```